draft-ietf-httpbis-digest-headers.md

```diff
@@ -318,6 +327,79 @@ Want-Digest: sha-256
 Want-Digest: sha-512;q=0.3, sha-256;q=1, unixsum;q=0
 ~~~
 
+
+# The Content-Digest Field {#content-digest}
+
+The `Content-Digest` field contains a comma-separated list of one or more content digest
```

Comment:
Since we're defining a new header here, can we just use structured fields for it entirely? So it'd be a dictionary with the field names as algorithms and the values as binaries.

Reply:
Excellent question. I like the idea but think that it would make it awkward because the algorithms define their encoding. So you'd have to carve out some exceptions and differences, which might catch implementers out.

Reply:
I wanted to switch to SF, but after a long discussion it was suggested to retain the current grammar,
even because SF do not support uppercase dictionary keys: this is a show stopper wrt being backward compatible.


Moreover  I don't agree wrt using different parsers  for `C-D` and  `D`: it is valuable that the same parser can process both fields.